assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -347,6 +349,7 @@ spec:
       - name: kms-kp
         secret:
           secretName: kp-wdek-secret
+          defaultMode: 0640
```

Comment:
defaultMode was already set to `0440` 2 lines down.

Reply:
thx John, good catch; fixed up now